docs/tenant.puml

```diff
@@ -0,0 +1,185 @@
+@startuml tenant
+skinparam DefaultTextAlignment center
+skinparam ArrowColor black
```

Comment:
Let's make arrow color blue or purple to better distinguish from other borders.

Reply:
well, I tried that, but now plantuml decided I get grayscale for every component unless I specify otherwise 🤦‍♂️

Reply:
alright - everything is recolored. It's ugly, but readable I think